tools/code-tools/jtreg.sh

```diff
@@ -50,20 +51,25 @@ buildJTReg()
     elif [ "$1" == "$JTREG_7" ]; then
       export JTREG_BUILD_NUMBER="1"
       export BUILD_VERSION="7"
-      export JAVA_HOME=/usr/lib/jvm/jdk-11.0.16.1+1
+      export JAVA_HOME=/usr/lib/jvm/jdk-11
```

Comment:
Are we confident this picks up the latest Java 11?  I assume we added a symlink here

Reply:
Looks like its been run successfully here: https://ci.adoptium.net/view/Dependencies/job/dependency_pipeline/781/ out of this PR branch.